be/src/fs/fs_broker.cpp

```diff
@@ -332,7 +332,7 @@ StatusOr<std::unique_ptr<WritableFile>> BrokerFileSystem::new_writable_file(cons
         return Status::NotSupported("Open with MUST_EXIST not supported by broker");
     } else if (opts.mode == CREATE_OR_OPEN) {
         if (auto st = _path_exists(path); st.ok()) {
-            return Status::NotSupported("Cannot write an already exists file through broker");
+            return Status::NotSupported("Cannot write an already exists file through broker, path="_format(path));
```

Comment:
Status::AlreadyExist

Reply:
opts.mode is CREATE_OR_OPEN, return error since broker doesn't support open an already exists file.